_data/exhibitions.yml

```diff
@@ -1,11 +1,5 @@
-- title: Firmenkontaktbörse University Jena
-  link: https://www.uni-jena.de/firmenkontaktboerse
-  date: 25. Nov 2020
+- title: jobwalk Jena
+  link: https://jena.jobwalk.city/
+  date: 10. Jul 2021
   time:
-  address: Online only
-
-- title: Jenaer Unternehmenstage
-  link: https://jenaer-unternehmenstage.de/
-  date: 26. Nov 2020
-  time:
-  address: Online only
\ No newline at end of file
+  address: Markt 1, 07743 Jena
```

Comment:
This formatting also works but usually we work with <br> (see e.g. https://github.com/ePages-de/epages-devportal/pull/589/files). Could you change the address accordingly? :)